addons/mail/static/src/components/drop_zone/drop_zone.scss

```diff
@@ -1,7 +1,12 @@
 .o_DropZone {
+    background-color: rgba($white, 0.75);
     border: 2px dashed;
     z-index: 1;
 
+    &.o_Chatter_dropZone {
```

Comment:
Do we really need a different style in this case? And if so, it should be moved into the chatter file instead and/or as a modifier field/class on the dropzone.

But referencing chatter in this file is not ok.